documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -157,10 +166,16 @@ With the SerializationModifier set to _Value,_ the payload is minimized to the f
 }
 ----
 
-TODO: Example where precision in value string to ValueOnly to value string is critical.
+If the property itself is the subject for ValueOnly the serialization is minimized to the following:
+[source,json,linenums]
+----
+{
+  5000
+}
+----
 
 
-For a _SubmodelElementCollection,_ the struct is serialized as objects denoted by curly brackets:
+For a _SubmodelElementCollection_, the struct is serialized within a submodel or structured submodel element as objects denoted by curly brackets:
```

Comment:
again, what is a **structured** submodel element?

Reply:
one that is no leaf element, might be inconsistent

Reply:
sorry, a structured element do have child elements. SMC and SML

Reply:
reformulated, structured submodel element not used any longer in text